site/documentation/content/api/event/index.md

```diff
@@ -134,3 +130,21 @@ The example below might be used by the MQTT adapter to indicate that a connectio
   }
 }
 ~~~
+
+### Device Provisioning Notification
+
+Device registries may send this event to indicate that a new device has been provisioned. This may be used as a hook for 
```

Comment:
FMPOV we should not tie this event to the gateway based auto-provisioning process here. The event can be used to convey the provisioning status of a device in a very generic sense. As such, it could also be used to indicate that a device has been de-provisioned (deleted) and it should also be possible to be used regardless of the trigger that caused the device's status to be changed, i.e. we could also send this event if the device had been created using the Device Registry Management API, right?

Reply:
Yes, if we intend to use the event in a more generic fashion that is correct.